pkg/GAPJulia/JuliaInterface/read.g

```diff
@@ -0,0 +1,29 @@
+#
+# JuliaInterface: Test interface to julia
+#
+# Reading the implementation part of the package.
+#
+ReadPackage( "JuliaInterface", "gap/JuliaInterface.gi");
+ReadPackage( "JuliaInterface", "gap/BindCFunction.gi" );
+
+dirs_julia := DirectoriesPackageLibrary( "JuliaInterface", "julia" );
+
+JuliaIncludeFile( Filename( dirs_julia, "gaptypes.jl" ) );
+
+dirs_libgap := DirectoriesPackageLibrary( "JuliaInterface", "../../../src" );
```

Comment:
This seems rather brittle to me. Can we ask Julia to resolve the path for us? It should "know" where `GAP.jl` is, shouldn't it?

(Also, I find the name `dirs_libgap` a little bit confusing -- to many things are referred to by libgap. Perhaps `dirs_libgap_jl` or `dirs_gap_jl`? Of course if we don't need them anymore, this becomes moot.)